core/loadpoint_api.go

```diff
@@ -316,6 +318,27 @@ func (lp *Loadpoint) SetDisableDelay(delay time.Duration) {
 	}
 }
 
+// GetBatteryBoost returns the battery boost
+func (lp *Loadpoint) GetBatteryBoost() bool {
+	lp.Lock()
+	defer lp.Unlock()
+
+	return lp.batteryBoost
+}
+
+// SetBatteryBoost sets the battery boost
+func (lp *Loadpoint) SetBatteryBoost(enable bool) {
+	lp.Lock()
+	defer lp.Unlock()
+
+	lp.log.DEBUG.Println("set battery boost:", enable)
```

Comment:
We should do a mode check and limit this to `minpv` (and maybe `pv`) in the first run. Otherwise it's getting complicated to visualize this. How should "off + boost" look like in the UI?

Reply:
...meaning the function needs to be able to return an error or silently ignore?

Reply:
I'd prefer throwing an error which is passed to HTTP response (bad request). This makes it clearer for a pure API user.